app/controllers/organizer/session_types_controller.rb

```diff
@@ -48,7 +46,12 @@ def destroy
   private
 
   def set_session_type
-    @session_type = @event.session_types.find(params[:id])
+    if params[:id]
```

Comment:
This is a curious change.  What's the benefit of bring new and find logic together?